content/automate/ManageIQ/Infrastructure/VM/Transform/Import.class/__methods__/list_driver_isos.rb

```diff
@@ -0,0 +1,46 @@
+module ManageIQ
+  module Automate
+    module Infrastructure
+      module VM
+        module Transform
+          module Import
+            class ListDriverIsos
+              def initialize(handle = $evm)
+                @handle = handle
+              end
+
+              def main
+                values_hash = {}
+                values_hash[nil] = '-- select drivers ISO from list --'
```

Comment:
@matobet Seems like it would be better to not set any default message here.  Instead below if you do not have a provider you could provide an message like "-- Select Provider first--".  This "select drivers" message would move down into the current `else` block where you are listing the ISOs.